### ISSUE TITLE
Plugin Update False Flag?

### ISSUE THREAD

**Issue by striebwj** [ClassicPress/ClassicPress-Migration-Plugin#41]
There are some reports of the plugin saying that there is a new update to the migration plugin. Even though it is the latest version, it seems to only be happening after the switch from WP to CP.  

See the following forum links:
https://forums.classicpress.net/t/migrate-plugin-after-install-showing-up-as-needing-update/147
https://forums.classicpress.net/t/installed-plugins/145/6

**Comment by nylen** [comment]
Yeah I think that explains this particular report at least, but it is not possible that an upgrade would show up for the migration plugin.
